application/F3DStarter.cxx

```diff
@@ -1636,4 +1640,23 @@ void F3DStarter::AddCommands()
         this->LoadFileGroup(index);
       }
     });
+    interactor.addCommand("open_file_dialog", 
+      [this](const std::vector<std::string>&)
+      {
+         const char* file = getenv("CTEST_OPEN_DIALOG_FILE");
+         std::cout << file << std::endl;
+         if (!file)
+         {
+           file = tinyfd_openFileDialog( "Open File", ".", 0, nullptr, nullptr, false);
+         }
```

Comment:
One stupid trick to cover the line would be:
```suggestion
         const char* file = getenv("CTEST_OPEN_DIALOG_FILE") || tinyfd_openFileDialog( "Open File", ".", 0, nullptr, nullptr, false);
```

Reply:
That would work, but it might be better to have the openFiledialog obviously untested so that in the future its obvious what would and wouldn't be caught.

Reply:
I agree with @hant-hub .

Coverage is just a metric, no need to hide that certain part is not tested.

Reply:
I'll resolve this change, we can always add more testing if necessary